byway/src/main/java/com/google/sps/data/UserInfo.java

```diff
@@ -28,7 +28,7 @@
 /** A class to make a UserInfo type, containing an email, id and a list of trip IDs per user. */
 public final class UserInfo {
 
-  public static final String DATASTORE_ENTITY_KIND = "user";
+  public static final String DATASTORE_ENTITY_KIND = "userInfo";
```

Comment:
Is this a change you agreed to as a group? Renaming an entity kind means that any old entities already in datastore still have the old kind, so while this kind of change is safe early in development, it does have consequences if you do it later on when you already have entities with this kind. There's no hard requirement that the kind string exactly match the class name, so it would be fine to leave this as `"user"`.

Reply:
I changed both entities to align with the class name, but since it's not a hard req I changed both class entities back.

Fixed